package.json

```diff
@@ -52,6 +52,8 @@
   ],
   "devDependencies": {
     "@solid/community-server": "^5.1.0",
+    "@testing-library/react": "^14.0.0",
```

Comment:
Add some libraries to test behavior of react components. React Testing Library is the one recommended by React MUI. React Testing Library runs code on the DOM level. I.E. it renders everything down to the DOM document sent to users, completely removing react, and making tests interact directly with HTML elements.

The other UI testing library I've used is enzyme, which keeps things at the react component level. Tests can then interact more easily with the React components themselves.

IMO, React Testing Library takes a better approach, since it forces the tests closer to what the user will actually see.